projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -44,240 +48,245 @@
+  "gui.draconicevolution.dislocator.add": "添加新的位置",
+  "gui.draconicevolution.dislocator.add.info": "添加当前位置\n- 在选定位置的下方添加",
+  "gui.draconicevolution.dislocator.add_1": "+1",
+  "gui.draconicevolution.dislocator.add_16": "+16",
+  "gui.draconicevolution.dislocator.add_all": "全部",
+  "gui.draconicevolution.dislocator.add_bottom.info": "添加当前位置\n- 添加至列表底部",
+  "gui.draconicevolution.dislocator.add_top.info": "添加当前位置\n- 添加至列表顶部",
   "gui.draconicevolution.dislocator.delete.info": "删除",
-  "gui.draconicevolution.dislocator.double_click_name": "双击以重命名。",
-  "gui.draconicevolution.dislocator.drag_to_move": "按住并拖动来移动位置。",
-  "gui.draconicevolution.dislocator.edit_lock.info": "Toggle edit lock",
-  "gui.draconicevolution.dislocator.fuel": "燃料：",
-  "gui.draconicevolution.dislocator.fuel.info": "The total number of teleports remaining.",
-  "gui.draconicevolution.dislocator.fuel_add_1.info": "Add 1 Ender Pearl from your inventory.",
-  "gui.draconicevolution.dislocator.fuel_add_16.info": "Add 16 Ender Pearls from your inventory.",
-  "gui.draconicevolution.dislocator.fuel_add_all.info": "Add all Ender Pearls from your inventory.",
-  "gui.draconicevolution.dislocator.mode_blink": "Use Mode:",
-  "gui.draconicevolution.dislocator.mode_tp": "Use Mode:",
-  "gui.draconicevolution.dislocator.must_unlock": "(must be unlocked)",
-  "gui.draconicevolution.dislocator.right_click_mode.info": "Switch between \"Teleport to Selected\" and \"Blink\" on right click.\\nYou can also configure keybindings.",
-  "gui.draconicevolution.dislocator.right_click_tp": "右击传送",
-  "gui.draconicevolution.dislocator.update": "设定这个",
-  "gui.draconicevolution.dislocator.update.info": "Update the selected location to your current coordinates.",
-  "gui.draconicevolution.draconium_chest.autofill_all": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_all.info": "Autofill All",
-  "gui.draconicevolution.draconium_chest.autofill_fill": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_fill.info": "Autofill Fill",
-  "gui.draconicevolution.draconium_chest.autofill_lock": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_lock.info": "Autofill Lock",
-  "gui.draconicevolution.draconium_chest.autofill_off": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.autofill_off.info": "Autofill Off",
-  "gui.draconicevolution.draconium_chest.color_picker": "更改颜色",
-  "gui.draconicevolution.draconium_chest.color_picker.info": "更改颜色",
-  "gui.draconicevolution.draconium_chest.smelt_energy": "Smelt Energy",
-  "gui.draconicevolution.draconium_chest.smelt_speed": "Smelt Speed",
-  "gui.draconicevolution.energy_net.bound_to_invalid": "工具被绑定到无效的方块上！",
+  "gui.draconicevolution.dislocator.double_click_name": "双击左键重命名。",
+  "gui.draconicevolution.dislocator.drag_to_move": "左键点击并拖拽移动。",
+  "gui.draconicevolution.dislocator.edit_lock.info": "切换编辑锁定",
+  "gui.draconicevolution.dislocator.fuel": "传送燃料：",
```

```suggestion
  "gui.draconicevolution.dislocator.fuel": "传送能量：",
```
主要考虑到它并不是“燃料”，当然和后面的统一就行

---

```suggestion
  "gui.draconicevolution.dislocator.fuel": "传送耗材：",
```